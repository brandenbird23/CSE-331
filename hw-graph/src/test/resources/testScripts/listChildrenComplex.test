# listChildrenComplex test file
# tests if listChildren stays consistent

# create a graph
CreateGraph graph1

# create a node
AddNode graph1 node1
AddNode graph1 node2

# create an edge
AddEdge graph1 node1 node1 edge1
AddEdge graph1 node1 node1 edge2
AddEdge graph1 node1 node2 edge3
AddEdge graph1 node1 node2 edge4

# print the nodes in the graph
ListNodes graph1
ListChildren graph1 node1