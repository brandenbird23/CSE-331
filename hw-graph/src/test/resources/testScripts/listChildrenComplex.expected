# listChildrenComplex test file
# tests if listChildren stays consistent

# create a graph
created graph graph1

# create a node
added node node1 to graph1
added node node2 to graph2

# create an edge
added edge edge1 from node1 to node1 in graph1
added edge edge2 from node1 to node1 in graph1
added edge edge3 from node1 to node2 in graph1
added edge edge4 from node1 to node2 in graph1

# print the nodes in the graph
graph1 contains: node1 node2
the children of node1 in graph1 are: node1(edge1) node1(edge2) node2(edge3) node2(edge4)